app/pages/browse-solutions/template.njk

```diff
@@ -7,7 +7,7 @@
   <div class="nhsuk-width-container" data-test-id="browse-solutions">
     <div class="nhsuk-u-padding-top-5" data-test-id="go-back-link">
       {{ backLink({
-        "href": "/",
+        "href": "/" | link,
```

Comment:
I think it's better if we didn't have this. It's something that someone can easily forget to do and it will cause bugs going forward. Can we try and resolve this issue without this please